src/entity_ref.rs

```diff
@@ -135,26 +136,64 @@ impl<'a, T: Component> Ref<'a, T> {
     }
 }
 
-unsafe impl<T: Component> Send for Ref<'_, T> {}
-unsafe impl<T: Component> Sync for Ref<'_, T> {}
+unsafe impl<T: ?Sized + Sync> Send for Ref<'_, T> {}
```

Comment:
I've marked the `Send` impl for `Ref` as requiring its wrapped value to be `Sync`, as it is easy to clone `Ref`s and alias the underlying value, so it is only safe to send a `Ref` across a thread if `T` is `Sync`.